common/utils/src/main/resources/error/error-classes.json

```diff
@@ -463,6 +463,12 @@
     ],
     "sqlState" : "42704"
   },
+  "COLLATION_SUPPORT_DISABLED" : {
```

Comment:
How about renaming this to `COLLATION_SUPPORT_NOT_ENABLED`?

Somehow it feels better: we want to convey the information that the feature is not enabled, rather than that it is disabled.

Reply:
Yeah, makes more sense.